app/gui2/package.json

```diff
@@ -124,13 +124,16 @@
     "sql-formatter": "^13.0.0",
     "tailwindcss": "^3.2.7",
     "tar": "^6.2.0",
+    "ts-loader": "^9.5.1",
     "tsx": "^4.7.1",
     "typescript": "~5.2.2",
     "unbzip2-stream": "^1.4.3",
     "vite": "^4.4.9",
     "vite-plugin-inspect": "^0.7.38",
     "vitest": "^1.3.1",
     "vue-react-wrapper": "^0.3.1",
-    "vue-tsc": "^1.8.27"
+    "vue-tsc": "^1.8.27",
+    "webpack": "^5.91.0",
```

Comment:
We should be using either vite or esbuild for bundling. Webpack is a huge dependency that is redundant with other tools we are already using.

Reply:
I [used webpack](https://github.com/JaroslavTulach/graaljs-websocket/commit/265bd0c064bcaf328d242f8bb013d494fba3b945) to get a single `.js` file to execute. [Graal.js](https://github.com/oracle/graaljs) implements [ECMAScript 2023 specification](https://262.ecma-international.org/14.0/), but _mostly just_ the [specification](https://262.ecma-international.org/14.0/). 

[Graal.js](https://github.com/oracle/graaljs) has issues supporting _commonjs modules_. There is [some commonjs modules support](https://github.com/oracle/graaljs/blob/master/docs/user/Modules.md#commonjs-modules-cjs), but I had problems to get it used together with ESM ([y.js](https://github.com/yjs/yjs) seems to have a mixture of commonjs and ES modules). I even asked on Dec 22, 2023:

>  Is there any chance the Y.js server could rewrite all its modules from CommonJs to EcmaScript modules?

Loading ESM would be way easier (as the behavior is set in the  [spec](https://262.ecma-international.org/14.0/)). 

However for the purposes of the [websocket PoC](https://github.com/JaroslavTulach/graaljs-websocket) I gave up and let [webpack do all the module resolution work](https://github.com/JaroslavTulach/graaljs-websocket/commit/265bd0c064bcaf328d242f8bb013d494fba3b945).

Anything that generates single `.js` file is OK. Alternatively using only ESM modules is also an option. Your guidance is welcomed, @Frizi.